layers/meta-balena-isg-503/recipes-core/images/balena-image.inc

```diff
@@ -11,4 +11,4 @@ IMAGE_CMD_balenaos-img_append () {
 
```

Comment:
@RockSupport , can you please confirm that when booted from SD-CARD, does the eMMC enumerate as /dev/mmcblk1 :+1: 
https://github.com/balena-os/balena-isg/blob/master/layers/meta-balena-isg-503/recipes-support/resin-init/resin-init-flasher.bbappend#L2

Reply:
Hi @RockSupport , can you please confirm if the device has an eMMC storage block and that it enumerates as /dev/mmcblk1 ? Thank you

Reply:
The main storage is emmc and  it enumerates as /dev/mmcblk1